DirectProgramming/DPC++/OpenCLInterop/src/dpcpp_with_opencl_objects.dp.cpp

```diff
@@ -0,0 +1,127 @@
+//==============================================================
+// Copyright © 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <stdio.h>
+#include <CL/sycl.hpp>
+#include "CL/opencl.h"
+using namespace sycl;
```

Comment:
Please verify this was run through clang-format.

Reply:
Yes this was run through clang-format